BraintreeCore/src/main/java/com/braintreepayments/api/ConfigurationCache.kt

```diff
@@ -0,0 +1,56 @@
+package com.braintreepayments.api
+
+import android.content.Context
+import androidx.annotation.VisibleForTesting
+import java.util.concurrent.TimeUnit
+
+internal class ConfigurationCache @VisibleForTesting internal constructor(
```

Comment:
```suggestion
internal class ConfigurationCache @VisibleForTesting constructor(
```
🧶 : This is a small nit, take or leave it. We'll get a second pass on a lot of the codebase once every thing's in Kotlin too.